waspc/examples/todoApp/todoApp.wasp

```diff
@@ -116,5 +116,10 @@ job mySpecialJob {
   perform: {
     fn: import { foo } from "@ext/jobs/bar.js",
     options: {=json { "retryLimit": 1 } json=}
+  },
+  schedule: {
+    cron: "*/5 * * * *",
+    performFnArg: {=json { "foo": "bar" } json=},
+    options: {=json { "retryLimit": 0 } json=}
```

Comment:
Everyone ok with the new `schedule` syntax?

Reply:
Sure, I can update the naming to something easier to grok for args. 👍🏻 In this case, there is no relationship between `perform.options` and `schedule.options`. I tried to keep them scoped to where they applied. Do you think it should interact? If so, I can make it.

Right now, you cannot schedule twice. If we want to, I could make it an array. But I think that would probably be an unlikely scenario atm. The escape hatch is just several instances if they really wanted to.

Reply:
Yeah, in thinking about it more, having `perform.options` be the default base, and then `schedule.options` override those does make the most sense to me. I will update it thnx

Reply:
Multiple schedules -> ok, if you think one is enough we can go with that! If people ask for more we can easily adjust.

perform.options as bas and schedule.options as addition -> sounds good to me!